templates/scenario/contact/create-contact.yaml

```diff
@@ -0,0 +1,83 @@
+# The system name that uniquely identifies the jig
+title: Create Contact
+# Description of the jig. This is not a required field and can be omitted.
+description: Use this jig to quickly and securely add a new contact to Dynamic Data.
+# The jig type used to display or capture data
+type: jig.default
+# icon that displays on the widget on the home hub
+icon: person
+
+header:
+  type: component.jig-header
+  options:
+    height: small
+    children:
+      type: component.image
+      options:
+        source:
+          uri: https://builder.jigx.com/assets/images/captureinfo.jpg
+
+# The controls that will be displayed on the jig are defined under the children node on a default jig   
+children:
+  # All input controls are placed on a form component
+  - type: component.form
+    # A form is uniquely identified by its instance id
+    instanceId: contactInfo
+    options:
+      children:
+        # To display two controls next to each other, they are added as children of a field-row component
+        - type: component.field-row
+          options:
+            children:
+              # A text-field component is used to capture text information on a form
+              - type: component.text-field
+                # A control is uniquely identified by its instance id
+                instanceId: firstName
+                options:
+                  label: First Name
+              - type: component.text-field
+                instanceId: lastName
+                options:
+                  label: Last Name
+        - type: component.text-field
+          instanceId: jobTitle
+          options:
+            label: Title
+        - type: component.text-field
+          instanceId: companyName
+          options:
+            label: Company
+        - type: component.text-field
+          instanceId: phone
+          options:
+            label: Phone
+            icon: phone
+        - type: component.email-field
+          instanceId: email
+          options:
+            label: Email
+            icon: email
+
+# The top level action on a default jig places a button at the bottom of the screen
+actions:
```

Comment:
I'm not sure how the flow of this template should work. 
In my opinion it should start with create jig ( so when using the the rest of the jigs we have at least one record in the database), then continue to view jig, then edit and then list. In the list I would add recommendation to create more records and add an option to do so.

Reply:
As a next step I think we should have a simple one pager docs that explains the scenario template and have a csv or json file to download with instructions to upload via manage. Please don't hold this back we can add this at some point

Reply:
Okay, great. We'll update the onSuccess messages and order of jig blocks and I'm approving the PR